contracts/bonding/BondingVotes.sol

```diff
@@ -0,0 +1,534 @@
+    }
+
+    /**
+     * @dev Checkpoints by account (delegators and transcoders).
+     */
+    mapping(address => BondingCheckpointsByRound) private bondingCheckpoints;
+    /**
+     * @dev Total active stake checkpoints.
+     */
+    TotalActiveStakeByRound private totalStakeCheckpoints;
+
+    /**
+     * @dev Modifier to ensure the sender is BondingManager
+     */
+    modifier onlyBondingManager() {
+        _onlyBondingManager();
+        _;
+    }
+
+    /**
+     * @notice BondingVotes constructor. Only invokes constructor of base Manager contract with provided Controller address
+     * @param _controller Address of Controller that this contract will be registered with
+     */
+    constructor(address _controller) Manager(_controller) {}
+
+    // IVotes interface implementation.
+    // These should not access any storage directly but proxy to the bonding state functions.
+
+    /**
+     * @notice Returns the name of the virtual token implemented by this.
+     */
+    function name() external pure returns (string memory) {
+        return "Livepeer Stake";
+    }
+
+    /**
+     * @notice Returns the symbol of the token underlying the voting power.
+     */
+    function symbol() external view returns (string memory) {
+        return livepeerToken().symbol();
+    }
+
+    /**
+     * @notice Returns the decimals places of the token underlying the voting.
+     */
+    function decimals() external view returns (uint8) {
+        return livepeerToken().decimals();
+    }
+
+    /**
+     * @notice Clock is set to match the current round, which is the checkpointing
+     *  method implemented here.
+     */
+    function clock() public view returns (uint48) {
+        return SafeCast.toUint48(roundsManager().currentRound());
+    }
+
+    /**
+     * @notice Machine-readable description of the clock as specified in EIP-6372.
+     */
+    // solhint-disable-next-line func-name-mixedcase
+    function CLOCK_MODE() public pure returns (string memory) {
+        return "mode=livepeer_round";
+    }
+
+    /**
+     * @notice Returns the current amount of votes that `_account` has.
+     */
+    function getVotes(address _account) external view returns (uint256) {
+        return getPastVotes(_account, clock());
+    }
+
+    /**
+     * @notice Returns the amount of votes that `_account` had at a specific moment in the past. If the `clock()` is
+     * configured to use block numbers, this will return the value at the end of the corresponding block.
+     * @dev Keep in mind that since this function should return the votes at the end of the _round (or timepoint in OZ
+     * terms), we need to fetch the bonding state at the next round instead. That because the bonding state reflects the
+     * active stake in the current round, which is the snapshotted stake from the end of the previous round.
+     */
+    function getPastVotes(address _account, uint256 _round) public view returns (uint256) {
```

Comment:
FTR of an offline discussion: The ERC5805 specifies that the `getPast...` functions should only be callable with a past timepoint value, so that its results can always be immutable (compared to `getVotes` that returns votes in current round [1]).

So we decided to make that change here and add an additional revert in case this is called for the current round. State from the current round 

[1] This behavior feels even more aligned with the change to return the stake in the end of the round instead of the start. The `getVotes` **is** actually mutable now, which it wasn't when we were returning the stake at the beginning of the round.